### PR TITLE
Image refresh for rhel-8-5

### PR DESCRIPTION
**Pull request by cockpituous**
Image refresh for rhel-8-5
 * [x] image-refresh rhel-8-5
 * [x] bump c-machines testlib to pick up https://github.com/cockpit-project/cockpit/commit/f754e5f72dcd08c8d9ac519c73046de3e42655a2: https://github.com/cockpit-project/cockpit-machines/pull/201
 * [x] fix "Not connected" toast alert on cockpit selinux page: https://github.com/cockpit-project/cockpit/pull/15932
 * [x] rebase after #2108 to fix failing cockpituous test
 * [x] Fix TestGrafana: https://github.com/cockpit-project/cockpit/pull/15934

### CODE REVIEW

**Comment by mvollmer** [comment]
I have filed https://bugzilla.redhat.com/show_bug.cgi?id=1968371 for the selinux gore. Let's see if we have to ignore the messages.

**Comment by martinpitt** [comment]
I'm afraid we do have to ignore the messages -- our naughty system is not built for that massive volume ("every test fails")

**Comment by mvollmer** [comment]
> I'm afraid we do have to ignore the messages -- our naughty system is not built for that massive volume ("every test fails")

The alternative would be to wait for the fix, but yeah, that doesn't really achieve much except not having to clean up our journal exceptions later.

**Comment by mvollmer** [comment]
The message ignoring will not take until https://github.com/cockpit-project/cockpit/pull/15906 has landed.

File: machine/machine_core/machine.py [hunk 1]
```diff
@@ -230,6 +230,10 @@ def allowed_messages(self):
             # but we have to ignore that general header line
             allowed.append('^Stack trace of.*')
 
+        if self.image in ['rhel-8-5']:
```
Comment (martinpitt) [inline machine/machine_core/machine.py:233]: Needs "distropkg" as well: [log](https://logs.cockpit-project.org/logs/pull-2085-20210610-053945-815f67ea-rhel-8-5-distropkg-cockpit-project-cockpit/log.html#90)

Comment (martinpitt) [reply machine/machine_core/machine.py:233]: And for some reason [does not work for c-machines](https://logs.cockpit-project.org/logs/pull-2085-20210610-053939-815f67ea-rhel-8-5-cockpit-project-cockpit-machinesn/log.html#5)

Comment (martinpitt) [reply machine/machine_core/machine.py]: Fixed for distropkg, sent https://github.com/cockpit-project/cockpit-machines/pull/201 for c-machines, tracking in todo list now.

**Comment by martinpitt** [comment]
There's some [SELinux regression](https://logs.cockpit-project.org/logs/pull-2085-20210610-065213-f4b8478e-rhel-8-5-cockpit-project-cockpit/log.html#119-2), the page now [shows](https://logs.cockpit-project.org/logs/pull-2085-20210610-065213-f4b8478e-rhel-8-5-cockpit-project-cockpit/TestSelinux-testTroubleshootAlerts-rhel-8-5-127.0.0.2-2601-FAIL.png) an additional "not connected" error.

**Comment by martinpitt** [comment]
This is an actual bug on the SELinux page. If connecting to setroubleshootd takes a while, this "Not connected" alert pops up and never goes away. I'll work on a cockpit fix.

**Review by martinpitt — APPROVED** [review]

**Pull request merged**